examples/C++/install_portaudio.sh

```diff
@@ -23,8 +23,6 @@ MACOS=`uname 2>/dev/null | grep Darwin`
 if [ -z "$MACOS" ]; then
   ./configure --without-jack --without-oss \
     --with-alsa --prefix=`pwd`/install --with-pic || exit 1;
-  sed -i '40s:src/common/pa_ringbuffer.o::g' Makefile
-  sed -i '40s:$: src/common/pa_ringbuffer.o:' Makefile
```

Comment:
Removing those lines will cause issues for Unix build. I'm going to merge it and fix.